chain/evm/compass.go

```diff
@@ -1048,6 +1099,7 @@ func (t *compass) GetSkywayEvents(
 		WithField("logs", len(logs)).
 		Debug("Ready to parse events")
 
+	var events []chain.SkywayEventer
```

Comment:
Unused?

Reply:
It is used below on 1133, to accumulate events, and it's returned at the end.